pyfstat/mcmc_based_searches.py

```diff
@@ -1698,11 +1758,16 @@ def export_samples_to_disk(self):
             fmt=outfmt,
         )
 
-    def _get_twoF_from_loglikelihood(self, idx=None):
+    def _get_detstat_from_loglikelihood(self, idx=None):
+        """Inverts the extra terms applied in logl()."""
         if idx is None:
-            return (self.lnlikes - self.likelihoodcoef) * 2
+            return (
+                self.lnlikes - self.likelihoodcoef
+            ) / self.likelihooddetstatmultiplier
         else:
-            return (self.lnlikes[idx] - self.likelihoodcoef) * 2
+            return (
+                self.lnlikes[idx] - self.likelihoodcoef
+            ) / self.likelihooddetstatmultiplier
```

Comment:
Reminder: I'm not sure, but maybe this can be simplified

Reply:
As in, get rid of the if-else?

Reply:
Ja! Behold!
```
return (self.lnlikes[idx or ...] - self.likelihoodcoef) / self.likelihooddetstatmultiplier
```
Where `...` is the `Ellipsis` object which takes everything.

Reply:
Nice! Want to add it as a suggestion or your own commit?

Reply:
f96d742

Reply:
Let me eat my words: `idx or ...` fails if `idx == 0`.

Reply:
`[... if idx is None else idx]` should work, at least on np arrays, which I think `self.nlikes` should be...?

Reply:
a10981d